pages/index.js

```diff
@@ -1,5 +1,22 @@
 import Link from "next/link";
+import styled from "styled-components";
+import Flex from "@/components/Layout/Flex";
+
+const StyledLink = styled(Link)`
+  color: var(--primary-title-color);
+  text-decoration: none;
+  font-size: 1.5rem;
+  &:hover {
+    background-color: darkgrey;
+  }
+`;
 
 export default function HomePage() {
-  return <Link href="./marketplace">Welcome to the Star Wars Bazar!</Link>;
+  return (
+    <Flex justifyContent="center">
+      <StyledLink href="./marketplace">
```

Comment:
As Diego mentioned in his previous review:

```suggestion
      <StyledLink href="/marketplace">
```